app.rb

```diff
@@ -24,8 +26,17 @@ class MakersBNB < Sinatra::Base
   end
 
   post '/spaces/new' do
-    Space.create(name: params[:name], description: params[:description], price: params[:price], checkin: params[:checkin], checkout: params[:checkout])
-
+    user = session[:user]
+    space = Space.create(
+      name: params[:name],
+      description: params[:description],
+      price: params[:price],
+      checkin: params[:checkin],
+      checkout: params[:checkout],
+      user_id: user.id
+    )
+    p space
```

Comment:
Good to remove these before committing.

Reply:
Good spot, I've removed them and recommitted